Models/MovieDetails.swift

```diff
@@ -26,10 +26,32 @@ struct MovieDetails: Codable {
         case runtime
         case overview
     }
+    
+    init(from decoder: Decoder) throws {
+        let values = try decoder.container(keyedBy: CodingKeys.self)
+        
+        title = try values.decode(String.self, forKey: .title)
+        
+        let posterPathString = try values.decode(String.self, forKey: .posterPath)
```

Comment:
Same as previous comment, from [here](https://developers.themoviedb.org/3/movies/get-movie-details) `poster_path` can be null so:
```suggestion
        let posterPathString = try values.decodeIfPresent(String.self, forKey: .posterPath)
```